metacat-main/src/main/java/com/netflix/metacat/main/services/notifications/sns/SNSNotificationServiceImpl.java

```diff
@@ -349,11 +349,13 @@ private UpdateOrRenameTableMessageBase createUpdateorRenameTableMessage(
                     new UpdatePayload<>(oldTable, patch)
                 );
             } else {
+                final QualifiedName newName = currentTable == null ? null : currentTable.getName();
                 return new RenameTableMessage(
                     id,
                     timestamp,
                     requestId,
                     name.toString(),
+                    newName == null ? "" : newName.toString(),
```

Comment:
Why not set it to null if null?

Reply:
Setting to null instead of empty string if null would be fine as well. If defaulting to null instead of empty string here helps on the client side then we can make that change.